controllers/keystoneapi_controller.go

```diff
@@ -291,7 +291,7 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 		jobDef,
 		keystonev1.DbSyncHash,
 		instance.Spec.PreserveJobs,
-		5,
+		time.Duration(5)*time.Second,
```

Comment:
there is a job.NewJob call at L385 as well

Reply:
and there is a NewDeployment call at L573

Reply:
there is GetDataFromSecret calls in L84 here and in keystoneservice_controller L431

Reply:
Fixed these. also there was GetDataFromSecret in `api/v1beta1/keystoneapi.go` I had to fix...